rubygem-bastion/rubygem-bastion.spec

```diff
@@ -80,18 +76,8 @@ mkdir -p %{buildroot}%{gem_dir}
 cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
-mkdir -p %{buildroot}%{foreman_bundlerd_dir}
-
-cat <<GEMFILE > %{buildroot}%{foreman_bundlerd_dir}/bastion.rb
-gem 'bastion'
-gem 'less-rails'
-GEMFILE
-
-%foreman_precompile_plugin -s
 %foreman_bundlerd_file
-
-mkdir -p %{buildroot}%{foreman_dir}/public/assets
-ln -s %{foreman_assets_plugin} %{buildroot}%{foreman_dir}/public/assets/bastion
```

Comment:
I included the change in 3.0.1 PR (https://github.com/theforeman/foreman-packaging/pull/977) if you would prefer it go into a release of 3.0.0 let me know. I was just going with updating both in one go.